### PR TITLE
Refactor get links

### PR DESCRIPTION
**Pull request by guillemcordoba**
Closes https://github.com/holochain/scaffolding/issues/151

Depends on https://github.com/holochain/scaffolding/pull/154

### CODE REVIEW

**Comment by mattyg** [comment]
Heads up @guillemcordoba holochain-client-js doesn't actually define a `Link` type

**Comment by guillemcordoba** [comment]
Hehe we are riding the PR wave!

**Comment by guillemcordoba** [comment]
Waiting on https://github.com/holochain/holochain-client-js/issues/217 to be fixed

**Comment by ThetaSinner** [comment]
Is this ready to review?

**Comment by ThetaSinner** [comment]
Oh I see I was just invited to review so yes, will take a look

File: src/scaffold/link_type/coordinator.rs [hunk 1]
```diff
@@ -141,29 +143,41 @@ fn get_links_handler_to_agent(
         .to_string(&Cardinality::Vector)
         .to_case(Case::Snake);
 
-    format!(
-        r#"#[hdk_extern]
-pub fn get_{plural_snake_to_entry_type}_for_{singular_snake_from_entry_type}({from_arg_name}: {from_hash_type}) -> ExternResult<Vec<AgentPubKey>> {{
-    let links = get_links({from_arg_name}, LinkTypes::{pascal_link_type_name}, None)?;
-    
-    let agents: Vec<AgentPubKey> = links
+    let get_deleted_links_handler = match delete {
```
Comment (ThetaSinner) [inline src/scaffold/link_type/coordinator.rs:146]: tiny thing: I probably prefer `if`/`else` to match on a boolean.

**Review by ThetaSinner — APPROVED** [review]
One minor thing and it's tricky to follow all the templating details but I trust the tests to take care of that. The change looks good to me!

**Pull request merged**